---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,7 @@ LDIRT = config.log .dep config.status co
 	Logs/* built .census install.* install-dev.* install-lib.* *.gz
 
 LIB_SUBDIRS = include libmisc libacl
-TOOL_SUBDIRS = getfacl setfacl chacl m4 man doc po test examples package debian
+TOOL_SUBDIRS = getfacl setfacl chacl m4 man doc po test examples package
 
 SUBDIRS = $(LIB_SUBDIRS) $(TOOL_SUBDIRS)
 
